app/controllers/presentations_controller.rb

```diff
@@ -37,6 +37,14 @@ def update
     end
   end
 
+  def destroy
+    @presentation = Presentation.find(params[:id])
+    authorize @presentation
+    @presentation.destroy
```

Comment:
if you are not checking the return value of an AR CRUD method, you should probably use its bang method instead.

`destroy!` will raise an error if the record fails to save.